hw_diag/diagnostics/env_var_diagnostics.py

```diff
@@ -19,6 +19,9 @@
 }, {
     'ENV_VAR': 'VARIANT',
     'DIAGNOSTIC_KEY': 'VA'
+}, {
+    'ENV_VAR': 'FIRMWARE_SHORT_HASH',
+    'DIAGNOSTIC_KEY': 'FH'
```

Comment:
`'DIAGNOSTIC_KEY': 'firmware_short_hash'` would be better

Reply:
Done.